onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4105,10 +4107,10 @@ def aten_select_scatter(self: TensorType, src: TensorType, dim: int, index: int)
     raise NotImplementedError()
 
 
-def aten_selu(self: TensorType) -> TensorType:
+def aten_selu(self: TFloat) -> TensorType:
     # selu(Tensor self) -> Tensor
 
-    raise NotImplementedError()
+    return op.Selu(self)
```

Comment:
Why do we put the logic like this within ONNX Script? As we discussed before, the logic relative to PyTorch specifically should be left in PyTorch exporter side, right?

Reply:
More of (2) should be in the function lib, so only the glue logic (not expressed by functions) lives in the exporter. We should aim for minimal op logic on the exporter (but allow full control at the same time so changes can be made independently)

Reply:
About "minimal op logic on the exporter", may I know the reason why we are doing this? What's the difference between calling these troch_aten ops and onnx scripts ops from PyTorch exporter?